### ISSUE TITLE
Use own embeddings?

### ISSUE THREAD

**Issue by mlmu** [KlugerLab/SpectralNet#16]
Hi! First of all - many thanks for your great code!
Could you please tell, whether it is possible to use your system with own embeddings? 
If yes, how could I do it in a proper way?
I tried it already with pre-trained paragraph2vec vectors, however I probably did smth. wrong, since I am always getting the error with Cholesky decomposition (Cholesky decomposition was not successful. The input might not be valid.) Embeddings which I use have dimensionality of 500. Have you any idea, why it happens? 

Many thanks in advance!

**Comment by lihenryhfl** [comment]
Hi, of course, thanks for using our code!

I couldn't tell from your question, but do you have an initial embedding dimensionality of 500, or a SpectralNet embedding dimensionality of 500? Also, how many classes do you have (or expect to have) in your dataset, and how balanced are these classes?

I'd recommend trying three things, in this order:
1. Reducing the SpectralNet learning rate.
2. Increasing the SpectralNet batch size.
3. Reducing the dimensionality of the spectralnet embeddings.
If this doesn't help, please let me know.
